trulens_eval/examples/expositional/frameworks/llama_index/llama_index_hybrid_retriever.ipynb

```diff
@@ -26,7 +26,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# ! pip install trulens_eval==0.24.0 llama_index==0.10.11 openai pypdf torch sentence-transformers"
+    "# ! pip install trulens_eval==0.24.0 llama_index==0.10.11 llama-index-readers-file llama-index-llms-openai llama-index-retrievers-bm25 openai pypdf torch sentence-transformers"
```

Comment:
Add these to optional imports requirements too?

Reply:
Llama-Index has 100s of component packages now. We probably don't want to track them all as optional requirements?